spec/features/home_page_spec.rb

```diff
@@ -92,4 +92,51 @@
       expect(find(@alert)).to have_content(@sign_in_msg)
     end
   end
+
+  describe 'contact us' do
+    before(:each) do
+      @name_textbox      = 'input#name'
+      @email_textbox     = 'input#email'
+      @select_menu       = 'select#subject'
+      @message_textarea  = 'textarea#message'
+      @message           = 'Hello, this is is cool.'
+      @send_button_text  = 'Send'
+      HydrusMailer.stub_chain(:contact_message, :deliver_now).and_return('a mailer')
+    end
+
+    it 'should show the full form and then generate an email when a user fills in the contact form when not logged in' do
```

Comment:
It should?   When I see that I hear "maybe it will." Can we assert "It will show..." instead?

Reply:
fixed